### ISSUE TITLE
Outreach outdated

### ISSUE THREAD

**Issue by alaninmcr** [FAIRdom/fairdom.github.io#43]
All the content under outreach seems to be outdated. It seems "sad"

**Comment by smza** [comment]
Yes the only the presentations are up dated. The rest is stuff that we carried over and it was decided this stuff needs to be kept
